---
layout: default
---
[NCBI BLAST](https://blast.ncbi.nlm.nih.gov/Blast.cgi)
<br />
[MSA Tools](https://www.ebi.ac.uk/Tools/msa/)
<br />
[BioPython](https://biopython.readthedocs.io/en/latest/Tutorial/chapter_align.html#alignment-tools)
<br />
*Monday (24/06/2019)*
<br />
* **Three things I learnt today:**
<br />
  * Learnt about the public databases for retrieving genomic and proteomic sequences.
  * Learnt how to find the complement of a gene sequence and how to reverse it in order to transcribe it to RNA using Python programming.
  * Learnt how to use machine learning approaches to identify the handwriting style of digits and name what number they are.
<br />
* **Three things I did today:**
<br />
  * Getting an introduction to what bioinformatics involves and being introduced to my supervisors
  * Searching the different public online databases for genomic and proteomic sequences.
  * Downloaded anaconda and played around with the different packages.
<br />
* **Three questions:**
<br />
  * What are the machine learning approaches used to identify plants/diseases/cancers?
  * How can python be used for data visualisation and what packages are used for them?
  * How many complete genomes of organisms have been found?
* **Paragraph:**
<br />
*Tuesday (25/06/2019)*
<br />
* **Three things I learnt today:**
<br />
  * Learn how to use the public websites that are capable of multiple sequence alignments such as MAFFT and BLAST.
  * Learnt how to use Support Vector Machine (SVM), Random Forest and Logistic Regression for handwritten digit classifications on python.
  * Learnt how to use Python to create sequence alignments.
<br />
* **Three things I did today:**
<br />
  * Went for a tour of the Monash Antibody Technologies Facility to observe the process of producing antibodies and immortal cells
  * Attended a presentation on how fluorescent proteins can be injected into organisms and be used to identify specific parts of a cell.
  * Discovered different public websites that can be used for multiple sequence analysis.
<br />
* **Three questions:**
<br />
  * How can python be used to create phylogenetic trees and analyse multiple sequence alignments?
  * What type of files and formats are used for multiple sequence analyses on python?
  * How can multiple sequence alignments be used to detect similarities and evolutionary changes?
* **Paragraph:**
<br />
*Wednesday (26/06/2019)*
<br />
* **Three things I learnt today:**
<br />
  *
  *
  *
<br />
* **Three things I did today:**
<br />
  *
  *
  *
<br />
* **Three questions:**
<br />
  *
  *
  *
<br />
* **Paragraph:**
<br />
*Thursday (27/06/2019)*
<br />
* **Three things I learnt today:**
<br />
  *
  *
  *
<br />
* **Three things I did today:**
<br />
  *
  *
  *
<br />
* **Three questions:**
<br />
  *
  *
  *
<br />
* **Paragraph:**
<br />
*Friday (28/06/2019)*
<br />
* **Three things I learnt today:**
<br />
  *
  *
  *
<br />
* **Three things I did today:**
<br />
  *
  *
  *
<br />
* **Three questions:**
<br />
  *
  *
  *
<br />
* **Paragraph:**
<br />
